tests/MilkCheckTests/EngineTests/ServiceGroupTest.py

```diff
@@ -1152,17 +1160,9 @@ def test_inter_subservice_deps_multi_level(self):
             'services': {
                 'subgroupA': {
                     'services': {
-                        'svcA':
```

Comment:
why removing this service? Was it making the test fail? or this is just for simplification?